target_iceberg/sinks.py

```diff
@@ -57,9 +57,20 @@ def process_batch(self, context: dict) -> None:
 
                 records: pa.Table = pa.Table.from_pylist(context["records"])
 
+                records = _remove_columns(records, schema)
```

Comment:
I believe it would be nice to let the user decide if they want to remove the columns or receive an exception. We could add a configuration property for this. What do you think?

Reply:
@ndrluis I added a new boolean property called `filter_columns`. If True, the columns will be removed.